### PR TITLE
Add Noto CJK (in slices of HK/TC/SC/JP/KR) and Noto Sans UI

### PR DESCRIPTION
**Pull request by nathan-williams**
Adding these files upstream will provide an upstream connection for fonts that can be found on fonts.google.com

### CODE REVIEW

**Review by marekjez86 — APPROVED** [review]
nitpicking:
Sans copyright inside the fonts has 2014-2020 copyright range (not 2015)
Serif copyright inside the fonts has 2017 copyright (not 2015)
I don't know if this matters. Otherwise looks good

**Comment by marekjez86** [comment]
nitpicking:
Sans copyright inside the fonts has 2014-2020 copyright range (not 2015)
Serif copyright inside the fonts has 2017 copyright (not 2015)
I don't know if this matters. Otherwise looks good (approved)

**Review by marekjez86 — APPROVED** [review]
LGTM

File: ofl/notosansjp/README [hunk 1]
```diff
@@ -0,0 +1,4 @@
+The font files here are not the original Noto Sans JP files. The files here have been created by subsetting the original files down to the Google Fonts Latin + Japanese subset. This is done in an effort to keep the dynamic subsetting times for these fonts at a reasonable level.
+
+Latin Subset: //depot/google3/java/com/google/fonts/backend/unicoderange/latin_unique-glyphs.nam
+Japanese Subset: //depot/google3/java/com/google/fonts/backend/unicoderange/japanese_unique-glyphs.nam
```
Comment (rsheeter) [inline ofl/notosansjp/README:4]: Based on this being here, search for google3 in * and fix everywhere. I think perhaps README files shouldn't be copied out and you should squash the log to eliminate it fromhistory.

File: ofl/notoserifjp/README [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+google.com/get/noto for more information.
+
+Built on 2017-10-24 from the following noto repository:
+-----
+Repo: noto-cjk
+Tag: v2017-06-01-serif-cjk-1-1
+Date: 2017-09-20 09:49:40 GMT
+Commit: 32a5844539f2e348ed36b44e990f9b06d7fb89fe
+
+Update serif CJK to 1.1.
```
Comment (rsheeter) [inline ofl/notoserifjp/README:11]: Again I think perhaps you don't want to include the READMEfiles

**Review by rsheeter — DISMISSED** [review]
Generally LGTM, just need to fix google3 refs

**Comment by thlinard** [comment]
I take this PR to ask a question that has been puzzling me for a while: these fonts exist in TTF VF, is there a reason to release them in OTF?

Thanks to the person who will take the time to answer!

File: ofl/notosansjp/README (hunk 1 above)
Comment (nathan-williams) [reply ofl/notosansjp/README]: README files removed from PR and commit history :)

File: ofl/notoserifjp/README (hunk 2 above)
Comment (nathan-williams) [reply ofl/notoserifjp/README]: Thanks for flagging.

**Comment by nathan-williams** [comment]
Hi Thomas, didn't want you to feel ignored... though I'm not very tuned into the history behind that question. I'm also interested in the answer. Going to ask around.

**Comment by davelab6** [comment]
While https://github.com/googlefonts/noto-cjk/tree/main/Sans/Variable/TTF has been around for 3 months, which might feel like a long time out here in the wilds of github, that's "less than a quarter" in corporate planning cycles, and moving from CFF statics to TTF VFs is an API upgrade that has to be done carefully. I requested the TTF VFs from Adobe because we have internal clients who can't process CFF at all, and the long term plan is to move over.

**Comment by marekjez86** [comment]
@thlinard : this is the first step to ensure consistency ... CJK VF is a longer and a separate project

**Comment by thlinard** [comment]
@davelab6 @marekjez86 
Thanks for the explanation!

**Comment by rsheeter** [comment]
For context, the CFF versions have been live for public use since 2018 (https://www.unicodeconference.org/presentations-42/S5T3-Sheeter.pdf) and get a reasonable amount of traffic (https://fonts.google.com/analytics). The traffic volume makes the static cff => variable ttf flip somewhat high risk, high reward.

Initially they were loaded in a manner that bypasses this repo for an experiment; had that experiment failed they would not have been offered to the public. The experiment passed and they have been public on fonts.google.com for some time; it's purely human error (...probably mine) they were not reflected here until recently.

**Pull request merged**